alf/utils/data_buffer.py

```diff
@@ -132,8 +132,6 @@ def _create_buffer(tensor_spec):
                     num_environments, dtype=torch.int64))
             self._buffer = alf.nest.map_structure(_create_buffer, data_spec)
             bs = BatchSquash(batch_dims=2)
```

Comment:
bs is not used. And BatchSquash imported at the beginning can be removed.

Reply:
good point